<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Browser</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QProgressBar {
        border: 2px solid #5A5A5A;
        border-radius: 5px;
        text-align: center;
    }
    QProgressBar::chunk {
        background-color: #4CAF50;
        width: 10px;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <item>
     <widget class="QListView" name="listView">
      <property name="maximumWidth">
       <number>150</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentWidget">
      <layout class="QVBoxLayout">
       <item>
        <widget class="QListWidget" name="imageGridWidget">
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="gridSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QListWidget::item {
               border: 2px solid transparent; /* 默认无边框 */
               padding: 2px;
           }
           QListWidget::item:selected {
               border: 2px solid #87CEEB; /* 淡蓝色边框 */
               background: rgba(135, 206, 235, 50); /* 淡蓝色背景，半透明 */
           }
          </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="visible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="infoWidget">
         <layout class="QVBoxLayout" name="infoLayout">
          <item>
           <widget class="QLabel" name="pathLabel">
            <property name="text">
             <string>路径: 未选择图片</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="creationTimeLabel">
            <property name="text">
             <string>创建时间: 未选择图片</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sizeLabel">
            <property name="text">
             <string>尺寸: 未选择图片</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
</ui>
